ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -501,6 +501,37 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                               seriesLength:(NSInteger)seriesLength
                                    options:(ORKPredefinedTaskOption)options;
 
+
+typedef NS_ENUM(NSUInteger, ORKValidatedSurveyType) {
+    ORKValidatedSurveyTypeNottinghamEADL,
+    ORKValidatedSurveyTypeStrokeImpactScale,
+    ORKValidatedSurveyTypeEpworthSleepScale,
+    ORKValidatedSurveyTypeIQCDE,
+    ORKValidatedSurveyTypeRivermeadMobilityIndex
+};
+
+
+/**
+ Returns one of a number of prebuilt validated survey tools:
+ 
+ The Nottingham EADL Scale is a standardised survey for measuring function in terms of which daily activities a participant can undertake.
+ The Stroke Impact Scale is a more comprehensive survey looking at the impact of neurological injury on an individual.
+ The Epworth Sleepiness Scale is a validated survey tool which screens for underlying sleep disorders (such as obstructive sleep apnoea).
+ The IQCDE is a validated survey for assessing longterm cognitive decline (to be completed by a long-term friend or relative).
+ The Rivermead Mobility Index is a validated survey quantifying everyday mobility.
+ 
```

Comment:
Agreed (and there are some explanations within the task files). But the documentation here is what appears on the documentation panel when you're writing the code in the first place (ie. when you create or click on the initialiser function), whereas the information in the actual survey task files isn't displayed.